oozie-to-airflow/run-sys-test

```diff
@@ -412,19 +412,21 @@ function verify_arguments {
 
 function header {
     echo
-    printf '#%.0s' {1..140}
+    SEPARATOR_WIDTH=${COLUMNS:=140}
```

Comment:
It does not work for me. I had to do "export COLUMNS" before I run the script to get the COLUMNS variable available for run-sys-tests script.

I think better solution will be to use $(tput cols). It's portable and always works.

Reply:
Thanks for suggestion. [SOA #1](https://pl.wikipedia.org/wiki/Standardowa_odpowied%C5%BA_administratora), but I updated the PR.